src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,21 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = [];
+
+const booksSlice = createSlice({
+  name: 'books',
+  initialState,
+  reducers: {
+    addsABook(state, action) {
+      const newState = state.push(action.payload);
+      return newState;
+    },
+    removesABook(state, action) {
+      const newState = state.filter(!action.payload);
```

Comment:
- [ ] I will encourage you to review this filter as I doubt if this would give what you and I expect. Consider having changing it to something like:
```
state.books = state.books.filter((book) => book.id !== action.id)
```

Remember to put the books initialState in an object too.